C/services/south-plugin-interfaces/python/python_plugin_interface.cpp

```diff
@@ -611,15 +611,15 @@ static void logErrorMessage()
 	//Get error message
 	PyObject *pType, *pValue, *pTraceback;
 	PyErr_Fetch(&pType, &pValue, &pTraceback);
+	PyErr_NormalizeException(&pType, &pValue, &pTraceback);
```

Comment:
I think, we should handle pTraceback as well. and log with verbose setting on/off? if not null

Reply:
pValue and pTraceback can be returned as NULL in some cases. pType is always non-NULL. And if (some of) these 3 returned values are not NULL, returned references are owned by this code, they have to be released eventually using PyErr_Restore. Check here: https://docs.python.org/3/c-api/exceptions.html

Reply:
Traceback can be handled more easily from within Python code if we use logger.exception level in error messages as exception level adds traceback to the messages by design.